database/migration/databaseMethods.ts

```diff
@@ -224,6 +227,32 @@ const databaseMethods = {
       }
     }
   },
+  updateFileSizes: async () => {
+    const files = (
+      await DBConnect.query({
+        text: `SELECT id, archive_path, file_path FROM file
+        WHERE file_size IS NULL;`,
+      })
+    ).rows
+
+    for (const file of files) {
+      const fileId = file.id
+      const archivePath = file.archive_path ?? ''
+      const filePath = file.file_path
+      const fullPath = path.join(FILES_FOLDER, archivePath, filePath)
+      try {
+        const size = (await fs.stat(fullPath)).size
+
+        await DBConnect.query({
+          text: `UPDATE file SET file_size = $1
+          WHERE id = $2`,
+          values: [size, fileId],
+        })
+      } catch {
+        console.log('Problem updating file size for:', fullPath)
+      }
+    }
+  },
```

Comment:
This is not really part of this PR, and not strictly required, but I find it useful for inspecting the file table.

Reply:
Closes #1067